contracts/staking/contracts/src/staking_pools/MixinCumulativeRewards.sol

```diff
@@ -264,59 +260,108 @@ contract MixinCumulativeRewards is
     )
         internal
         view
-        returns (uint256)
+        returns (uint256 reward)
     {
-        // sanity check inputs
         if (memberStakeOverInterval == 0) {
             return 0;
         }
 
-        // sanity check interval
-        if (beginEpoch >= endEpoch) {
+        // Sanity check interval
+        if (beginEpoch > endEpoch) {
```

Comment:
Can this ever actually trigger? We use an assertion for a similar check in `_forceSetMostRecentCumulativeRewardEpoch`

Reply:
If the code is written correctly, none of these conditions should ever be true. The thing is, it's super easy to make changes to the accounting that accidentally trips one, which is why I think it's valuable to have assertions in place. These gave me a ton of insight when I was working on this PR. But since these are essentially just for our benefit, maybe we should replace them with plain string reverts?

Reply:
I'd be fine with that personally.